spec/features/reviews/01_index_review_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+feature 'when user goes to park show page, park reviews are listed', %{
+  As a user
+  I want to see park reviews
+  So that I can read or view reviews
+  } do
+  # ACCEPTANCE CRITERIA
+  # * If I go to a park's show page, I can view all of its reviews
+
+  scenario 'user views reviews on park show page' do
+    user = create(:user)
+    park = create(:park)
+    review = create(:review, user: user, park: park)
```

Comment:
Delete lines 12-13. Just create a review with `review = create(:review)` and it will automaticallycreate a user and a park as well. Then you can do,
```
park = review.park
```